pkg/liquidity-mining/contracts/gauges/base/BaseRootGauge.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/SafeERC20.sol";
+
+import "../StakelessGauge.sol";
+
+interface IL1StandardBridge {
+    function depositERC20To(
+        address _l1Token,
+        address _l2Token,
+        address _to,
+        uint256 _amount,
+        uint32 _l2Gas,
+        bytes calldata _data
+    ) external;
+}
+
+contract BaseRootGauge is StakelessGauge {
+    using SafeERC20 for IERC20;
+
+    // Unlike on Optimism, on Base we do not need to maintain a dynamic gas limit in the factory to pass along
+    // on calls to the bridge.
+    //
+    // The gas limit on L2 is computed based in part upon the _MIN_GAS_LIMIT passed in on the bridge call.
+    // However, this value has a "floor" of RELAY_CONSTANT_OVERHEAD, set to 200_000. Since this floor is already
+    // twice the gas actually required to simply mint BAL, we can just pass in 0 for `_minGasLimit`, and the full
+    // computation will yield a total gas limit of around 288k.
+    //
+    // This parameter is provided for non-standard bridge tokens, or other unforeseen edge cases where more gas
+    // might be required. We do not need it for our simple, conforming token.
+    // solhint-disable-next-line max-line-length
+    // See https://github.com/ethereum-optimism/optimism/blob/595d5916e568ee4fcff5cb8d236a05361a50a881/packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol#L331
+    uint32 private constant _MIN_GAS_LIMIT = 0;
```

Comment:
:+1: sounds reasonable.

These bridges usually support performing actions on the L2, which is unnecessary for a simple erc20 transfer.